Source/WSTagView.swift

```diff
@@ -25,6 +29,46 @@ open class WSTagView: UIView, UITextInputTraits {
             setNeedsDisplay()
         }
     }
+    
+    open var imagePlaceholder = UIImage(systemName: "person.circle.fill") {
```

Comment:
Could you add a screenshot of what this looks like, please? Do we always want to show an image?